### PR TITLE
Bump @bufferapp/ui version to 5.36.0

### PR DESCRIPTION
**Pull request by MayaUribe**
## Description

Bump @bufferapp/ui version to 5.36.0

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/campaign-form/components/CampaignForm/index.jsx [hunk 1]
```diff
@@ -148,6 +148,7 @@ const CampaignForm = ({
               : t('campaigns.campaignForm.createTitle')}
           </Headline>
           <Input
+            id="campaignName"
```
Comment (MayaUribe) [inline packages/campaign-form/components/CampaignForm/index.jsx:151]: I wanted to use the new prop already 😬

Comment (anafilipadealmeida) [reply packages/campaign-form/components/CampaignForm/index.jsx:151]: You might need to update the snapshots then! If you'd like you can also delete the aria-required and aria-label here, or leave it for another time 😅

Comment (anafilipadealmeida) [reply packages/campaign-form/components/CampaignForm/index.jsx:151]: I forgot we no longer have snapshots here 😂

Comment (MayaUribe) [reply packages/campaign-form/components/CampaignForm/index.jsx:151]: I still "updated" the snapshots I forgot too 🤣 I'll delete those fields 🙌

**Pull request merged**